api-schema/src/main/java/org/stellar/anchor/api/shared/StellarPayment.java

```diff
@@ -18,19 +19,24 @@ public class StellarPayment {
 
   Amount amount;
 
+  @JsonProperty("payment_type")
```

Comment:
Can we stick with gson?

Reply:
We are mixing Gson and Jackson (in Spring and Kafka) throughout the project. This bug is one example where using both caused us some pain. Using Gson here would mean writing a new SerDe for it, which may get thrown away if we move everything to Jackson.

I want to get this patch release out so we can unblock our partners, but I'll create a Jira to address this serialization issue long term.